qemu/tests/getfd.py

```diff
@@ -52,7 +52,7 @@ def has_fd(pid, filepath):
         path = os.path.join(fdfiles_dir, name)
         fd = os.open(path, os.O_RDWR | os.O_CREAT)
         response = vm.monitor.getfd(fd, name)
-        os.close(fd)
+        vm.monitor.closefd(fd, name)
```

Comment:
If you closed the file descriptor for the qemu process then the following check should be failed.
```python
        if not has_fd(pid, path):
            raise error.TestError("QEMU process does not seem to have a file "
                                  "descriptor pointing to file %s" % path)
```